### ISSUE TITLE
`Automerge 1.0 API` changes impact `slate-collaborative` behaviour

### ISSUE THREAD

**Issue by raphael10-collab** [cudr/slate-collaborative#80]
Automerge 1.0` dramatically change the `API`:

https://github.com/automerge/automerge

https://github.com/automerge/automerge/blob/main/CHANGELOG.md

Thus, what has been deployed here has to be upgraded accordingly

**Comment by cudr** [comment]
Yes, I know about this change. But thinking, I need to do some deep research in light of https://josephg.com/blog/crdts-go-brrr/

**Comment by jlarmstrongiv** [comment]
Automerge 2.0 is released 🎉 https://automerge.org/blog/automerge-2/

It would also be nice to be agnostic to the network layer https://automerge.org/docs/cookbook/real-time/

**Comment by raphael10-collab** [comment]
Interesting. Thank you @jlarmstrongiv
